### ISSUE TITLE
Fail to build with WebKit on windows

### ISSUE THREAD

**Issue by ericLemanissier** [copperspice/copperspice#175]
Building on windows copperspice 1.7.1 from sources downloaded from https://download.copperspice.com/copperspice/source/ `WITH_WEBKIT=ON` errors with:
```
CMake Error at src/webkit/CMakeLists.txt:1 (add_library):
  Cannot find source file:
    (...)/copperspice/src/3rdparty/webkit/Source/WebCore/plugins/win/PaintHooks.asm
```

**Comment by bgeller** [comment]
We did wee your comment on an older CS commit. Please note that comments on commits can be hard to find and this is not the best way to raise and issue.  This might be why you posted a separate issue. 

This file is of course available on github and we will update the CMake ignore list for consistency and to support MSVC users. Let's try to understand that changes like this are easy to make and will happen once someone brings it to our attention.

**Comment by ericLemanissier** [comment]
Well, yes, this is exactly why I reported this issue. I'm glad we understand each other.

**Comment by agserm** [comment]
Glad you understand comments on commits are not a good idea and an issue only needs to be reported in one place, preferably as a ticket for tracking.

**Comment by ericLemanissier** [comment]
100% off topic, but you started it: I'm not sure to understand why you're repeating that it's better to report an issue, considering I reported an issue before you both advised me to do so. If you were offended by me comment on the commit, I am very sorry because it was not my intention. If you think my comment on the commit was inefficient, I realized it when I got no answer, so this is why I reported this issue. If you think I was somehow pissed when I reported this issue, then I apologize, because this is not the case.
